airbyte-container-orchestrator/src/main/java/io/airbyte/container_orchestrator/orchestrator/DbtJobOrchestrator.java

```diff
@@ -77,11 +77,10 @@ public Optional<String> runJob() throws Exception {
         Math.toIntExact(jobRunConfig.getAttemptId()),
         workerConfigs.getResourceRequirements(),
         new DbtTransformationRunner(
-            processFactory, NormalizationRunnerFactory.create(
-                destinationLauncherConfig.getDockerImage(),
+            processFactory, new DefaultNormalizationRunner(
                 processFactory,
-                NormalizationRunnerFactory.NORMALIZATION_VERSION,
-                destinationLauncherConfig.getNormalizationDockerImage())));
+                destinationLauncherConfig.getNormalizationDockerImage(),
```

Comment:
Shouldn't we be passing the `normalizationTag` here?

Reply:
For the `IntegrationLauncherConfig` class, I added a `normalizationDockerImage` field that already contains the name of the normalization repository and its tag.